### ISSUE TITLE
Broken on React Native >= 0.60.0 (Invariant Violation: Element type is invalid)

### ISSUE THREAD

**Issue by toumas** [tomLadder/react-native-echarts-wrapper#27]
WebView was finally removed from react-native in [v0.60](https://facebook.github.io/react-native/blog/#lean-core-removals) thus this package no longer works with the newer versions of react-native.

[Here's](https://github.com/toumas/rnew_repro) the issue reproduced.

The error:
```
Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `ECharts`.

This error is located at:
    in RCTView (at View.js:35)
    in View (at src/index.js:158)
    in ECharts (at App.js:25)
    in RCTView (at View.js:35)
    in View (at App.js:24)
    in App (at renderApplication.js:40)
    in RCTView (at View.js:35)
    in View (at AppContainer.js:98)
    in RCTView (at View.js:35)
    in View (at AppContainer.js:115)
    in AppContainer (at renderApplication.js:39)
```

**Comment by tomLadder** [comment]
Hey @toumas.

Thank you for reporting this issue. I'm currently rewriting some internals. The upcoming release will fix this problem.

Thanks,
Tom

**Comment by tomLadder** [comment]
just to be sure.

did you install `react-native-webview`?

**Comment by toumas** [comment]
In the application I'm developing it is included in dependencies. In the repro it is not but the issue is the same.

**Comment by tomLadder** [comment]
Just released [v1.4.3](https://www.npmjs.com/package/react-native-echarts-wrapper/v/1.4.3) 🎉.
This release adds support for RN 0.60
